src/components/atoms/InfinitScroll.tsx

```diff
@@ -1,34 +1,35 @@
 import React, { FC, useEffect, useCallback, useRef } from 'react';
 import { makeStyles } from '@material-ui/core';
+import { useStore } from '../../store/storeConfig';
 
 function isScrollEnd(element: HTMLDivElement) {
-  return (element.scrollTop + element.clientHeight) === element.scrollHeight;
+  return element.scrollTop + element.clientHeight > element.scrollHeight - 30;
```

Comment:
Do not use magic numbers. put it as a const at the top of the file (with a proper name, like "INIFINTE_SCROLL_OFFSET"
![image](https://user-images.githubusercontent.com/11487010/98266225-b8bccf80-1f92-11eb-93dc-02dca451c53a.png)